packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership.jsx

```diff
@@ -66,7 +66,7 @@ export const ProofOfOwnership = observer(() => {
         return <POONotRequired />; // Client does not need proof of ownership.
     }
     if (status === POO_STATUSES.rejected) {
```

Comment:
Lets replace all `POO_STATUSES` with `AUTH_STATUS_CODES` from `@deriv/shared`

Reply:
@shaheer-deriv can you confirm on this?

Reply:
this could've been a surest thing in Dubai office today 🚀 , but sadly likhith already made this change in his PR https://github.com/binary-com/deriv-app/pull/10116/